Week-5/Ex3.js

```diff
@@ -0,0 +1,4 @@
+console.log(hasDuplicate([1,2,3,-1,3]));
+console.log(hasDuplicate([1,2,3,-1]));
+
+const hasDuplicate = (arr) => new Set(arr).size != arr.length
```

Comment:
optional chaining

Reply:
I am not sure about the scope of using the optional chaining (?.) operator.

Reply:
https://javascript.info/optional-chaining

Read about this here